cmd/log-validator/main_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/letsencrypt/boulder/test"
+	"testing"
+)
+
+func TestLineValid(t *testing.T) {
```

Comment:
nit: make this 3 functions for maximum clarity:
1) calls lineValid on a valid log line, asserts it doesn't error
2) calls lineValid on an invalid log line, asserts it does error
3) calls lineValid on an invalid log line, asserts nothing, calls lineValid on the resulting error, asserts it does error

Reply:
Did this. For case (3), I kept both asserts, because if calling lineValid on an invalid log line didn't result in an error, it doesn't make sense to them pass that non-error into lineValid again.